### PR TITLE
docs: added `known_transaction_details` clarification

### PR DESCRIPTION
**Pull request by onmax**
## What's in this pull request?

I have been playing with the web client and it has taken me longer than I would like to admit to work out what this parameter does.

If we add this clarification, developers will have better context when using this feature.

## Pull request checklist

- [x] All tests pass. The project builds and runs.
- [x] I have resolved any merge conflicts.
- [x] I have resolved all `clippy` and `rustfmt` warnings.

### CODE REVIEW

**Comment by sisou** [comment]
There's more changes coming with https://github.com/nimiq/core-rs-albatross/pull/2218.

Inspiration for docs can also come from the description of the original implementation: 
https://github.com/nimiq/core-js/blob/fad56cdf5a1cb13ca61f867b46048b0e83e85d94/src/main/generic/api/Client.js#L719-L839

Maybe this can become part of #2218?

**Comment by onmax** [comment]
Note apart: In the future we could use objects as parameters.

Is it hard to do? Maybe use function overriding? (not sure if it possible) 

I am thinking of something like this:

```ts
interface GetTransactionsByAddressOptions {
    sinceBlockHeight?: number,
    knownTransactionDetails?: PlainTransactionDetailsArrayType,
    limit? : number
    minPeers?: number
}
getTransactionsByAddress(address, options?: GetTransactionsByAddressOptions) {}
```

File: Cargo.lock [hunk 1]
```diff
@@ -3118,12 +3118,9 @@ dependencies = [
...
 [[package]]
 name = "log"
-version = "0.4.17"
+version = "0.4.21"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "abb12e687cfb44aa40f41fc3978ef76448f9b6038cad6aef4259d3c095a2382e"
-dependencies = [
- "cfg-if",
-]
+checksum = "90ed8c1e510134f979dbc4f070f87d4313098b704861a105fe34231c70a3901c"
```
Comment (jsdanielh) [inline Cargo.lock]: This shouldn't be part of the PR since it's already in `albatross`. Did you miss to rebase `albatross`?

Comment (onmax) [reply Cargo.lock]: I rebased with github. I will fix it

Comment (onmax) [reply Cargo.lock]: Fixed

**Review by jsdanielh — APPROVED** [review]

**Review by hrxi — APPROVED** [review]
LGTM.
